### PR TITLE
COR-406 Core server Dockerfile

### PR DESCRIPTION
**Pull request by ludydoo**

### CODE REVIEW

File: build/package/core.Dockerfile [hunk 1]
```diff
@@ -1,29 +1,25 @@
...
+ADD . .
 
-# Build the application
-RUN go build -o main .
+RUN go mod verify
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o main .
 
 # Build a small image
-FROM scratch
+FROM golang:alpine
```
Comment (neb42) [inline build/package/core.Dockerfile:20]: Why do we need golang here? Can we not run the binary with go?

Comment (ludydoo) [reply build/package/core.Dockerfile:20]: Yeah, good catch. Had some troubles with compiling in sqlite binaries that we use in tests

**Pull request merged**